### PR TITLE
chore: update peer-ui package deps

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/8627/desktop-recent-files-share-button-not-disabled-for-pending-files

#### Testing instructions  
check that `npm install` gets the latest `peer-ui`...?

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
     "node-sass": "4.8.3",
     "npm-run-all": "4.1.2",
     "opt-cli": "1.6.0",
-    "peer-ui": "github:PeerioTechnologies/ui-library#master",
+    "peer-ui": "github:PeerioTechnologies/ui-library",
```
Comment (suprko) [inline package.json]: @seavan @anri-asaturov 

PeerUI has releases now
https://github.com/PeerioTechnologies/ui-library/releases

Is this line sufficient for `npm install` to get the latest release? do we need to publish to npm?

Comment (anri-asaturov) [reply package.json]: No need for npm, do it like this https://github.com/PeerioTechnologies/peerio-desktop/blob/master/app/package.json#L68

Comment (suprko) [reply package.json]: So we need to manually version bump in package.json each time?

Comment (anri-asaturov) [reply package.json]: Yes

**Comment by suprko** [comment]
@anri-asaturov version number added

**Pull request merged**